src/core/metstate_mod.F90

```diff
@@ -272,6 +275,11 @@ SUBROUTINE Zero_MetState( MetState, RC )
 
    END SUBROUTINE Zero_MetState
 
+   !>
+   !! \brief Allocate the MetState object
```

Comment:
I think you can put this at the start and it will be recognized as the brief automatically

Reply:
If you just write it puts it under a longer description and not a brief statement. I believe they treat them separately.

Reply:
Hmm, maybe that behavior is configurable...